### PR TITLE
docs: Update README with more examples

### PR DESCRIPTION
**Pull request by tsmithv11**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

[//]: # "
    # PR Title
    Be aware that we use the title to create changelog automatically and therefore only allow specific prefixes
    - break:    to indicate a breaking change, this supersedes any of the types
    - feat:     to indicate new features or checks
    - fix:      to indicate a bugfix or handling of edge cases of existing checks
    - docs:     to indicate an update to our documentation
    - chore:    to indicate adjustments to workflow files or dependency updates
    - platform: to indicate a change needed for the platform
    Additionally we encourage adding a scope to the prefix, which indicates the targeted framework, otherwise 'general' will be assumed.
    ex.
    feat(terraform): add CKV_AWS_123 to ensure that VPC Endpoint Service is configured for Manual Acceptance
"

## Description

Updated README with examples provided by Itai

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,6 +258,31 @@ kube-system namespace:
 checkov -d . --skip-check kube-system
 ```
 
+Run a scan of a container image:
+```sh
+checkov --framework sca_image --docker-image sha256:1234example --dockerfile-path /Users/path/to/Dockerfile --bc-api-key ... --repo-id test/test_docker
+```
+OR
+```sh
+checkov --docker-image <image-name>:tag --dockerfile-path /User/path/to/Dockerfile --bc-api-key ...
```
Comment (Saarett) [inline README.md:267]: the docker-image could also be the hash or the id of the image

File: README.md [hunk 2]
```diff
@@ -258,6 +258,31 @@ kube-system namespace:
 checkov -d . --skip-check kube-system
 ```
 
+Run a scan of a container image:
+```sh
+checkov --framework sca_image --docker-image sha256:1234example --dockerfile-path /Users/path/to/Dockerfile --bc-api-key ... --repo-id test/test_docker
```
Comment (Saarett) [inline README.md:263]: the repo-id is non mandatory if I recall correctly

File: README.md (hunk 1 above)
Comment (Saarett) [reply README.md:267]: Worth mentioning that if it's a local docker file, you should build it first, and then extract the hash/id of the image

**Review by Saarett — COMMENTED** [review]
Nice!! 🎖️

File: README.md (hunk 2 above)
Comment (tsmithv11) [reply README.md]: Good point. It's not. Removed.

File: README.md (hunk 1 above)
Comment (tsmithv11) [reply README.md:265]: Good idea. Updated.

**Review by Saarett — APPROVED** [review]

**Review by NoaAzoulay — APPROVED** [review]

**Review by gruebel — APPROVED** [review]

**Pull request merged**